tests/regex/test_verilog_repo.stdout.regex

```diff
@@ -0,0 +1,13 @@
+(?s)onefetch-committer-name ~ git version ([^\r\n]+)\r?\n(?x) # MATCH LINE 1 CAPTURE GIT VERSION
```

Comment:
```suggestion
onefetch-committer-name ~ git version ([^\r\n]+)\r?\n(?x) # MATCH LINE 1 CAPTURE GIT VERSION
```
I don't think the `s` flag is needed if you're explicitly matching `\n`. Also, instead of disabling and re-enabling `x` on every line, you might just want to change ` ` into `\ `, which should be matched instead of ignored even when using `x`. Not sure which would be more readable TBH :thinking:

Reply:
Ah, that is how to match spaces in x modified regex, yeah I couldn't figure out how to match spaces.  Thanks for explaining that aspect.

Reply:
yep path separator worked, I do not understand what the documentation is talking about.
[dfe994c](https://github.com/o2sh/onefetch/pull/827/commits/dfe994cc7dfb7edf7ef590ef948d8e913f748e08)